### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by lemmy** [pfeodrippe/tla-edn-module#1]
Are you going to create a release so that users can just grab a jar file?  For example, I don't have `clj` on my machine.

**Comment by pfeodrippe** [comment]
Hi @lemmy, sorry for the delayed response (I need to fix my notifications...).

Yeah, we could, do you mean a jar file in the `Releases` section of the Github? You can also add https://mvnrepository.com/repos/clojars as one of your maven repositories and copy the `Maven` section from https://clojars.org/io.github.pfeodrippe/tla-edn-module.

**Comment by lemmy** [comment]
It seems this warrants an installation guide for the average TLC user who isn't familiar with the JVM.

**Comment by pfeodrippe** [comment]
Added https://github.com/pfeodrippe/tla-edn-module/blob/master/README.adoc#installation-and-usage to ease a little bit the process of installation, let me know what you think o/

There is also the possibility of using `tladeps` to just return the classpath, making it composable for more advanced users and automatic tooling (using the option `--tladeps-raw-deps`, see https://github.com/pfeodrippe/tladeps#installation).

I'm assuming here that the user is running TLC from the terminal, not from the toolbox or VSCode extension.

**Comment by lemmy** [comment]
> I'm assuming here that the user is running TLC from the terminal, not from the toolbox or VSCode extension.

According to our execution statistics, this excludes the majority of TLC users.

**Comment by pfeodrippe** [comment]
> > I'm assuming here that the user is running TLC from the terminal, not from the toolbox or VSCode extension.
> 
> According to our execution statistics, this excludes the majority of TLC users.

Yeah, these tools already have a way to add things to the classpath, right? If so, people can use `tladeps` already by copying its classpath output.

Another alternative, if there are ways the user can set a completely different command (other than `java -cp ...`), it also work as `tladeps`  can receive exactly the same arguments as the original, e.g. `java -cp tla2tools.jar tlc2.TLC -help` turns into `tladeps tlc2.TLC -help`, so you can use TLC, or SANY or whatever we have in `tla2tools`.

A third alternative is to integrate ways to transparently install and use `tladeps` into the toolbox or the extension, but this requires some small PRs and convincing the owner that `tladeps` is worth it (which I don't know yet). It works at Unix systems, but it should work at Windows with no or little effort (`babashka`, the executable behind the scenes, works at Windows pretty well).

Which proposal (if not another) do you have in mind?
